src/util/address.rs

```diff
@@ -364,6 +364,40 @@ impl Address {
         };
         format!("{}:{:#}", schema, self)
     }
+
+    /// Parsed addresses do not always have *one* network. The problem is that legacy testnet,
+    /// regtest and signet addresse use the same prefix instead of multiple different ones. When
```

Comment:
nit (NOT MERGE BLOCKING)
/s/addresse/addresses

Reply:
I assume I'll not fix it to not invalidate reviews unless a major change request comes in.